docs/running/slurm.md

```diff
@@ -38,6 +87,116 @@ This is the default partition, and will be used when you do not explicitly set a
 
 The following sections will provide detailed guidance on how to use SLURM to request and manage CPU cores, memory, and GPUs in jobs. These instructions will help users optimize their workload execution and ensure efficient use of CSCS computing resources.
 
+## Affinity
+
+The following sections will document how to use Slurm on different compute nodes available on Alps.
+To demonstrate the effects different Slurm parameters, we will use a little command line tool [affinity](https://github.com/bcumming/affinity) that prints the CPU cores and GPUs that are assinged to each MPI rank in a job, and which node they are run on.
+
+We strongly recommend using a tool like affinity to understand and test the Slurm configuration for jobs, because the behavior of Slurm is highly dependent on the system configuration.
+Parameters that worked on a different cluster -- or with a different Slurm version or configuration on the same cluster -- are not guaranteed to give the same results.
```

Comment:
As discussed yesterday, I like the output of the affinity program. However, do you think it might make sense to add a collapsed info box briefly saying that one can get similar, but more basic, information by printing `CUDA_VISIBLE_DEVICES` and by setting `srun --cpu-bind=verbose`, though the information isn't as nicely readable. Simply as a quick and dirty alternative (that can also e.g. easily be enabled/added to existing batch scripts as a sanity check).

Reply:
I'm happy with the information that was added elsewhere, closing.